app/controllers/users_controller.rb

```diff
@@ -995,6 +995,26 @@ def destroy
     render json: success_json
   end
 
+  def ignore
+    raise Discourse::NotFound unless SiteSetting.ignore_user_enabled
+
+    ::IgnoredUser.find_or_create_by!(
+      user: current_user,
+      ignored_user: User.find_by(id: params[:ignored_user_id]))
+    render json: success_json
+  end
+
+  def follow
```

Comment:
Maybe this has been discussed before, but for me follow is not the opposite of ignore

Reply:
@jjaffeux how about `watch`?